hcloud/load_balancer_types/__init__.py

```diff
@@ -5,4 +5,11 @@
     LoadBalancerTypesClient,
```

Comment:
A `noqa` is still present for this import.

Reply:
Fixed, thanks!